dev/release/00-prepare-test.rb

```diff
@@ -184,6 +184,10 @@ def test_update_version_pre_tag
                      hunks: [
                        ["-version = \"#{@snapshot_version}\"",
                         "+version = \"#{@release_version}\""],
+                       ["-arrow = { path = \"../arrow\" }",
+                        "-parquet = { path = \"../parquet\" }",
+                        "+arrow = { version = \"#{@release_version}\" }",
```

Comment:
Ah, sorry.
Expected is `arrow = "#{@release_version}"`. We don't need `{ version = ... }`.

Reply:
Thank you for pointing out. I've fixed them.